caravel/assets/spec/javascripts/explore/components/QueryAndSaveBtns_spec.jsx

```diff
@@ -0,0 +1,37 @@
+var React = require('react');
+var { expect } = require ('chai');
+var { shallow } = require('enzyme');
+
+const QueryAndSaveButtons = require(
+  '../../../../javascripts/explore/components/QueryAndSaveBtns'
+);
+
+describe('QueryAndSaveButtons', () => {
+  let defaultProps = {
+    canAdd: 'True',
+    onQuery: () => {}
+  };
+
+  // It must render
+  it('renders', () => {
+    expect(React.isValidElement(<QueryAndSaveButtons {...defaultProps} />)).to.equal(true);
+  });
+
+  // Test the output
+  describe('output', () => {
+    let wrapper;
+
+    beforeEach(() => {
+      wrapper = shallow(<QueryAndSaveButtons {...defaultProps} />);
```

Comment:
hi @lelandrichardson! :wave: i'm just getting enzyme setup in this repo and i'm having trouble with `shallow` rendering. when i run this spec, i get the following error. any ideas? 

<img width="1280" alt="screenshot 2016-07-13 21 38 30" src="https://cloud.githubusercontent.com/assets/130878/16828162/36d23206-4942-11e6-88a0-542cb1beab70.png">